ruby_on_rails/app_initialisation.md

```diff
@@ -71,17 +80,10 @@ unless File.exist?('.git/hooks/pre-push')
 end
 ```
 
-* And the `commit-msg` hooks:
-
-```ruby
-unless File.exist?('.git/hooks/commit-msg')
-  system 'ln -s ../../bin/commit-msg .git/hooks/commit-msg'
-end
-```
-
 * Change also  `system!('bundle install')` to `system!('bundle install --jobs=3 --retry=3')`
+and uncomment the installation of yarn dependencies with `system('bin/yarn')`
 
-* Run `bundle exec rails db:migrate` to generate an empty `schema.rb` file
+* Run `bundle exec rails db:create db:migrate` to generate an empty `schema.rb` file
```

Comment:
is it actually necessary to run `db:migrate` ? wouldn't `db:setup` be enough?

Reply:
`db:setup` fails if there is no schema. `db:migrate` is enough.